app/src/main/java/com/android/partagix/model/FinishedLoansViewModel.kt

```diff
@@ -34,41 +34,44 @@ class FinishedLoansViewModel(db: Database = Database()) : ViewModel() {
   }
 
   fun getFinishedLoan() {
+    println("--------------finishedloan")
```

Comment:
Sorry but all `println` must be gone. Please replace it with a Log if you prefer to keep a log.